### PR TITLE
Reword: Use the same writing as in other component

### PR DESCRIPTION
**Pull request by yvanzo**
# Problem

A pair of confirmation messages were hardly shortened, using inappropriate typewriter apostrophe, needlessly duplicating messages to be translated.

# Solution

Use “Yes, I am sure” for all.

# Testing

Manually opened both pages locally using sample data.

### CODE REVIEW

File: root/admin/LockedUsernameUnlock.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const LockedUsernameUnlock = ({
       <form method="post" name="confirm">
         <FormCsrfToken form={form} />
         <FormSubmit
-          label="Yes, I’m sure"
+          label="Yes, I am sure"
```
Comment (reosarevok) [inline root/admin/LockedUsernameUnlock.js:38]: We might as well put this into l() then since it's an existing translation?

Comment (yvanzo) [reply root/admin/LockedUsernameUnlock.js:38]: It’s an admin form though, we shouldn’t really translate it.

Comment (reosarevok) [reply root/admin/LockedUsernameUnlock.js:38]: As you prefer :) I think we're very inconsistent right now with translating some and not others, so it'd be good to eventually drop those if we decide not to translate admin forms as a rule.

Comment (yvanzo) [reply root/admin/LockedUsernameUnlock.js:38]: Maybe we should have a dummy `l_admin` that does nothing but prevents messages from being added to POT files.

**Review by reosarevok — APPROVED** [review]

**Comment by yvanzo** [comment]
Selenium test failed with the usual unrelated async error, ignoring it as I tested the two pages manually.

**Pull request merged**